### DISCUSSION TITLE
文件和文件夹、坚果云同步问题

### DISCUSSION THREAD

**Discussion by FranticEr** [zotero-chinese/wiki#205] [求助 / Q&A] [answered]
我使用坚果云作为文件同步工具，根据网上教程将‘已连接附件的根目录’设置为D:\zotero_file，‘数据存储位置’设置为M:\zotero_file，有些文件没有同步到‘已连接附件的根目录’中去，这是为什么，这不应该是坚果云的问题吧。

**Comment by northword** [comment, marked as answer]
如果使用 坚果云，建议通过 WebDAV 配置同步，请参阅：https://zotero-chinese.com/user-guide/sync 。

链接的附件通常需要与 ZotFile 配合，我们已经不建议使用 ZotFile。
